docs/docs/cmd/viva/engage/engage-community-list.mdx

```diff
@@ -16,6 +16,25 @@ m365 viva engage community list [options]
 
 <Global />
 
+## Permissions
+
+<Tabs>
+  <TabItem value="Delegated">
+
+  | Resource        | Permissions                              |
+  |-----------------|------------------------------------------|
+  | Microsoft Graph | Community.Read.All                       |
```

Comment:
Much appreciated that you added these permission scopes. Let's just ensure that the table is properly formatted so it looks good in the console.
```suggestion

  | Resource        | Permissions        |
  |-----------------|--------------------|
  | Microsoft Graph | Community.Read.All |
```

Let's fix it for the other pages as well.